assets/javascripts/needleeditor.js

```diff
@@ -177,6 +177,11 @@ NeedleEditor.prototype.LoadTags = function(tags) {
   this.UpdateTextArea();
 };
 
+NeedleEditor.prototype.LoadProperty = function(property) {
+  this.needle.properties = property;
+  this.UpdateTextArea();
+}
```

Comment:
The semicolon after this assignment is missing. (Yes, semicolons are optional in JavaScript but we use them consistently.)